.github/workflows/qodana_code_quality.yml

```
@@ -17,12 +17,23 @@ jobs:
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.10)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)